src/emm/emm_cli.py

```diff
@@ -225,16 +225,23 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
 @cli.command(context_settings=dict(max_content_width=100))
 @click.option("--title", help="Title for the pull request.")
 @click.option("--body", help="Body for the pull request")
+@click.option(
+    "--remote",
+    default="origin",
+    help="Remote to push against for the pull request. The default would be 'origin'",
```

Comment:
[nit] "The default is 'origin'" or "[default=origin]". "would be" doesn't make a lot of sense in this context.